### ISSUE TITLE
Error importing fortecubeview

### ISSUE THREAD

**Issue by davidsousarj** [evangelistalab/fortecubeview#2]
Hi. I have been using fortecubeview for a few weeks in my work computer. Today I installed it in my personal computer as well, but when I try to import it I get a error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_13480/3433171198.py in <module>
     12 # Trying the python visualization tool
     13 from pyscf.tools import cubegen
---> 14 import fortecubeview
     15 
     16 for i in range(25):

~/.local/lib/python3.10/site-packages/fortecubeview/__init__.py in <module>
----> 1 from .fortecubeview import plot, vib, colorschemes, geom
      2 

~/.local/lib/python3.10/site-packages/fortecubeview/fortecubeview.py in <module>
----> 1 from .cube_viewer import *
      2 from .vib_viewer import *
      3 from .mol_viewer import *
      4 
      5 # The allowed color schemes

~/.local/lib/python3.10/site-packages/fortecubeview/cube_viewer.py in <module>
      7 
      8 import ipywidgets as widgets
----> 9 from .py3js_renderer import Py3JSRenderer
     10 
     11 found_forte = importlib.util.find_spec("forte") is not None

~/.local/lib/python3.10/site-packages/fortecubeview/py3js_renderer.py in <module>
      6 
      7 from IPython.display import display
----> 8 from pythreejs import *
      9 from .atom_data import *
     10 

~/.local/lib/python3.10/site-packages/pythreejs/__init__.py in <module>
      4 
      5 from .enums import *
----> 6 from .pythreejs import *
      7 from .traits import *
      8 from ._example_helper import *

~/.local/lib/python3.10/site-packages/pythreejs/pythreejs.py in <module>
     18 
     19 
---> 20 from .core.BufferAttribute import BufferAttribute
     21 from .core.Geometry import Geometry
     22 from .core.BufferGeometry import BufferGeometry

~/.local/lib/python3.10/site-packages/pythreejs/core/BufferAttribute.py in <module>
      2 
      3 from .._base.Three import ThreeWidget
----> 4 from .BufferAttribute_autogen import BufferAttribute as BaseBufferAttribute
      5 
      6 

~/.local/lib/python3.10/site-packages/pythreejs/core/BufferAttribute_autogen.py in <module>
     11 from .._base.uniforms import uniforms_serialization
     12 from ..enums import *
---> 13 from ..traits import *
     14 
     15 from .._base.Three import ThreeWidget

~/.local/lib/python3.10/site-packages/pythreejs/traits.py in <module>
     16 from ipywidgets import widget_serialization
     17 
---> 18 from ipydatawidgets import DataUnion, NDArrayWidget, shape_constraints
     19 
     20 import numpy as np

~/.local/lib/python3.10/site-packages/ipydatawidgets/__init__.py in <module>
      5 # Distributed under the terms of the Modified BSD License.
      6 
----> 7 from .ndarray import *
      8 from .widgets import DataWidget
      9 from ._version import __version__, version_info

~/.local/lib/python3.10/site-packages/ipydatawidgets/ndarray/__init__.py in <module>
      5 # Distributed under the terms of the Modified BSD License.
      6 
----> 7 from .media import DataImage
      8 from .serializers import array_serialization, data_union_serialization
      9 from .traits import NDArray, shape_constraints

~/.local/lib/python3.10/site-packages/ipydatawidgets/ndarray/media.py in <module>
     15 
     16 
---> 17 class DataImage(DataWidget, DOMWidget):
     18     """A data-widgets based Image widget.
     19     """

~/.local/lib/python3.10/site-packages/traitlets/traitlets.py in __init__(cls, name, bases, classdict)
    956         """Finish initializing the HasDescriptors class."""
    957         super().__init__(name, bases, classdict)
--> 958         cls.setup_class(classdict)
    959 
    960     def setup_class(cls, classdict):

~/.local/lib/python3.10/site-packages/traitlets/traitlets.py in setup_class(cls, classdict)
    979     def setup_class(cls, classdict):
    980         cls._trait_default_generators = {}
--> 981         super().setup_class(classdict)
    982 
    983 

~/.local/lib/python3.10/site-packages/traitlets/traitlets.py in setup_class(cls, classdict)
    971         for _, v in getmembers(cls):
    972             if isinstance(v, BaseDescriptor):
--> 973                 v.subclass_init(cls)
    974 
    975 

~/.local/lib/python3.10/site-packages/ipydatawidgets/ndarray/union.py in subclass_init(self, cls)
     57 
     58     def subclass_init(self, cls):
---> 59         cls._instance_inits.append(self.instance_init)
     60         return super().subclass_init(cls)
     61 

AttributeError: type object 'DataImage' has no attribute '_instance_inits'
```

**Comment by davidsousarj** [comment]
I have already tried uninstall the dependencies, clear `pip cache` and install them again, downgrading `scikit-image`to version 0.19 (current is 0.20), nothing worked.

**Comment by davidsousarj** [comment]
I did not get any response here, so I keep searching about the error in the internet. I have found [this](https://github.com/vidartf/ipydatawidgets/issues/57). So, it seems that it is an error in `pythreejs`, not in `fortecubeview`. I had to downgrade the package `ipydatawidgets`from the current version (4.3.3) to the previous:

`pip install ipydatawidgets==4.3.2`

After doing this, `fortecubeview` is working nicely.
